Wikipedia/Code/ArticleViewController.swift

```diff
@@ -632,9 +634,9 @@ class ArticleViewController: ThemeableViewController, HintPresenting, UIScrollVi
             let siteURL = self.articleURL.wmf_site,
                   let project = WikimediaProject(siteURL: siteURL) {
                 ArticleLinkInteractionFunnel.shared.logArticleView(pageID: pageID.intValue, project: project)
```

Comment:
I'm unsure about how we should treat article peek and pops. I missed this case when logging the original page views.

In testing, this area gets called once when long pressing > article preview is displayed. Tapping further to go into the full screen article does not log this again. Backing out of peek is not ideal either, because we logged it as a page view when really they didn't read the full view of the article.

Maybe a temporary measure (if product is ok with it) is to wrap this up in a conditional that I found:

```
if !isBeingPresentedAsPeek {
    {log page view}
}
```

So that we're only logging when they don't come in through the long press.

Reply:
Let's confirm with Product that it's ok for us to suppress this.